src/test-tree/createTests.ts

```diff
@@ -222,37 +228,64 @@ export class TestCase {
 		readonly package_name: string,
 		readonly proof_boolean: boolean,
 		readonly stubbing_request?: boolean,
+		readonly module_name?: string,
 	) {}
 
 	getLabel(): string {
 		return `${this.harness_name}`;
 	}
 
+	// Get the fully qualified name for a harness from the file name, module name and harness name
+	getFullyQualifiedName(): string {
+		if (this.module_name === '') {
+			const fileName = extractFileName(this.file_name);
+			if (fileName === 'main' || fileName === 'lib' || fileName === 'mod') {
+				return `${this.harness_name}`;
+			}
+			return `${fileName}::${this.harness_name}`;
+		} else if (this.module_name !== '') {
+			// expand on this logic
```

Comment:
What does that comment mean for someone that is reading the code for the first time?

Reply:
Ah, forgot to remove that, :P